web-client/src/app/views/pay/pay.page.ts

```diff
@@ -145,19 +148,22 @@ export class PayPage implements OnInit {
         }
```

Comment:
Do we need to add `this.autoLogoutService.enableAutoLogout()` in the catch (error) block above on line 144?

Reply:
No, it is not needed, will delete it in the latest commit.

Reply:
Wait, I think it's needed? I meant another catch block.
I saw you added the enableAutoLogout in the catch block and then I saw another one where you didn't add it.

Reply:
I don't think so as the one I deleted should not have been added in the catch block as it (enableAutologout() method) was already added on the try block.